iredis/config.py

```diff
@@ -57,6 +57,9 @@ def __setter__(self, name, value):
             self.queued_commands = []
         super().__setattr__(name, value)
 
+    def __getattr__(self, attr):
```

Comment:
Why need this?

Reply:
avoid AttributeError

Reply:
`AttributeError` is expected, `return None` will slice exceptions and make debug more difficult.